src/components/inline_edit/inline_edit_text.tsx

```diff
@@ -59,25 +52,13 @@ export const EuiInlineEditText: FunctionComponent<EuiInlineEditTextProps> = ({
     };
   }, [_readModeProps, isReadOnly]);
 
-  const formProps = {
-    sizes,
-    defaultValue,
-    placeholder,
-    inputAriaLabel,
-    startWithEditOpen,
-    readModeProps,
-    editModeProps,
-    isLoading,
-    isInvalid,
-    isReadOnly,
-  };
-
   return (
     <EuiInlineEditForm
       className={classes}
       css={cssStyles}
+      readModeProps={readModeProps}
+      sizes={sizes}
       {...rest}
```

Comment:
This is missing `isReadOnly`
```suggestion
      readModeProps={readModeProps}
      isReadOnly={isReadOnly}
      sizes={sizes}
      {...rest}
```